### PR TITLE
chore: migrate rollup-plugin-image

### PR DESCRIPTION
**Pull request by shellscape**
## Rollup Plugin Name: `@rollup/plugin-image`

This PR contains:
- [ ] bugfix
- [x] feature
- [ ] refactor
- [ ] documentation
- [x] other

Are tests included?
- [x] yes (*bugfixes and features will not be merged without tests*)
- [ ] no

Breaking Changes?
- [x] yes (*breaking changes will not be merged unless absolutely necessary*)
- [ ] no

- **Breaking:** Minimum compatible Rollup version is 1.20.0
- **Breaking:** Minimum supported Node version is 8.0.0
- **Breaking:** Plugin will generate a `const` variable for exporting the image by default. To obtain the old default functionality, use the `dom: true` option.

List any relevant issue numbers:

### Description

This PR migrates rollup-plugin-image and gives it some TLC. Dependencies have been updated, tests added, options defined and documented, adds a new `dom` option, and contains the breaking changes listed above.

### CODE REVIEW

File: packages/image/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# @rollup/plugin-image
```
Comment (NotWoods) [inline packages/image/README.md]: Should the README include badges at the top like [the other packages](https://raw.githubusercontent.com/rollup/plugins/master/packages/buble/README.md)?

File: packages/image/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+console.log(logo);
+```
+
+Create a `rollup.config.js` [configuration file](https://www.rollupjs.org/guide/en/#configuration-files) and import the plugin:
+
+```js
+import image from "@rollup/plugin-image";
+
+export default {
+  input: "src/index.js",
```
Comment (NotWoods) [inline packages/image/README.md]: Maybe use single quotes here to match the package code and other READMEs

File: util/test.js [hunk 3]
```diff
@@ -1,3 +1,10 @@
+const getCode = async (bundle) => {
```
Comment (NotWoods) [inline util/test.js]: `getCode` should be used from `testBundle`

File: packages/image/README.md (hunk 2 above)
Comment (NotWoods) [reply packages/image/README.md]: Nevermind, looks like other packages are switching

File: packages/image/README.md (hunk 1 above)
Comment (shellscape) [reply packages/image/README.md]: Yes, yes it should.

File: packages/image/README.md (hunk 2 above)
Comment (shellscape) [reply packages/image/README.md]: No that's actually an error with the MD linting from prettier. I'll look into it.

File: util/test.js (hunk 3 above)
Comment (shellscape) [reply util/test.js]: Not necessarily. Leaving it separate to evaluate usefulness.

File: package.json [hunk 4]
```diff
@@ -53,7 +53,7 @@
       "git add"
     ],
     "*.md": [
-      "prettier --write",
+      "prettier --single-quote --write",
```
Comment (wesleygrimes) [inline package.json]: Wondering if we should create a `.prettierrc` file for this? Not holding up approval for it, just a thought for future.

File: packages/image/README.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+## Requirements
+
+This plugin requires an [LTS](https://github.com/nodejs/Release) Node version (v8.0.0+) and Rollup v1.20.0+.
+
+## Install
+
+Using npm:
+
+```console
+npm install @rollup/plugin-dsv --save-dev
```
Comment (wesleygrimes) [inline packages/image/README.md]:
```suggestion
npm install @rollup/plugin-image --save-dev
```

File: packages/image/rollup.config.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import buble from 'rollup-plugin-buble';
```
Comment (wesleygrimes) [inline packages/image/rollup.config.js]: is this right? Should it be `buble`?

File: package.json (hunk 4 above)
Comment (shellscape) [reply package.json]: That'll screw with eslint-config-rollup. I'd like to not use a `.prettierrc` if we can help it.

File: packages/image/README.md (hunk 5 above)
Comment (shellscape) [reply packages/image/README.md]: noooo

File: packages/image/rollup.config.js (hunk 6 above)
Comment (shellscape) [reply packages/image/rollup.config.js]: Yeah that's verbatim from the old project

File: package.json (hunk 4 above)
Comment (wesleygrimes) [reply package.json]: got it.

Comment (wesleygrimes) [reply package.json]: we could also create some config file for this and pass the config file in the run command here if we end up adding more settings here.

Comment (shellscape) [reply package.json]: that'd be an option

**Pull request merged**